apps/palette-tauri/src/components/aurora.css

```
@@ -0,0 +1,507 @@
+  --aurora-highlight-strong: inset 0 1px 0 rgba(255, 255, 255, 0.055);
+  --aurora-active-glow: 0 0 0 1px color-mix(in srgb, #29b6f6 28%, transparent),
+                        0 2px 8px color-mix(in srgb, #29b6f6 18%, transparent);
+  --aurora-focus-ring: color-mix(in srgb, #29b6f6 30%, transparent);
+
+  /* Radii */
+  --aurora-radius-1: 14px;
+  --aurora-radius-2: 18px;
+  --aurora-radius-3: 22px;
+
+  /* Typography */
+  --aurora-font-display: 'Manrope', -apple-system, BlinkMacSystemFont, system-ui, sans-serif;
```

Stylelint (17.11.1)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)